egs/wsj/s5/steps/dur_model/make_nnet_config.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
```

Comment:
Actually I wanted to keep it simple for the first version. I have seen make_configs.py but I thought that was too much options and configurations for now.